src/patternfly/components/Content/styles.scss

```diff
@@ -1,36 +1,164 @@
+  // --pf-c-content--hero-title--MarginBottom: var(--pf-global--spacer--lg);
+  // update variables above when spacers are updated
+  // the following are spacers as defined in the issue
+  --pf-c-content--hero-title--MarginTop: 24px;
+  --pf-c-content--hero-title--MarginBottom: 16px;
+  --pf-c-content--hero-title--LineHeight: var(--pf-global--sm--LineHeight);
+  --pf-c-content--hero-title--FontSize: var(--pf-global--hero-title--FontSize);
+  --pf-c-content--hero-title--FontWeight: var(--pf-global--normal--FontWeight);
+
+  // h1 - Main title
+  // --pf-c-content--main-title--MarginTop: var(--pf-global--spacer--xxl);
+  // --pf-c-content--main-title--MarginBottom: var(--pf-global--spacer--md);
+  // update variables above when spacers are updated
+  // the following are spacers as defined in the issue
+  --pf-c-content--main-title--MarginTop: 24px;
+  --pf-c-content--main-title--MarginBottom: 16px;
+  --pf-c-content--main-title--LineHeight: var(--pf-global--sm--LineHeight);
+  --pf-c-content--main-title--FontSize: var(--pf-global--main-title--FontSize);
+  --pf-c-content--main-title--FontWeight: var(--pf-global--normal--FontWeight);
+
+  // h2 - Secondary title
+  // --pf-c-content--secondary-title--MarginTop: var(--pf-global--spacer--xl);
+  // --pf-c-content--secondary-title--MarginBottom: var(--pf-global--spacer--sm);
+  // update variables above when spacers are updated
+  // the following are spacers as defined in the issue
+  --pf-c-content--secondary-title--MarginTop: 24px;
+  --pf-c-content--secondary-title--MarginBottom: 16px;
+  --pf-c-content--secondary-title--LineHeight: var(--pf-global--sm--LineHeight);
+  --pf-c-content--secondary-title--FontSize: var(--pf-global--secondary-title--FontSize);
+  --pf-c-content--secondary-title--FontWeight: var(--pf-global--semi-bold--FontWeight);
+
+  // h3 - Sub-title
+  // --pf-c-content--sub-title--MarginTop: var(--pf-global--spacer--lg);
+  // --pf-c-content--sub-title--MarginBottom: var(--pf-global--spacer--md);
+  // update variables above when spacers are updated, and delete variable below
+  // the following are spacers as defined in the issue
+  --pf-c-content--sub-title--MarginTop: 24px;
+  --pf-c-content--sub-title--MarginBottom: 16px;
+  --pf-c-content--sub-title--LineHeight: var(--pf-global--md--LineHeight);
+  --pf-c-content--sub-title--FontSize: var(--pf-global--sub-title--FontSize);
+  --pf-c-content--sub-title--FontWeight: var(--pf-global--semi-bold--FontWeight);
+
+
+  // I'm not sure what to name these items, if we aren't trying to use elements in variable names
+  // I get why that rule is important for other components where the styles are associated with classes
+  // but this component is unique in that styles are associated with elements
+  // and it's extra confusing to try to remember what name is associated with what heading level
+
+  // h4
+
+  // h5
+
+  // h6
+
+  // should we define margin top for the following sizes?
```

Comment:
Ignore this question. I'm still waiting for styles for the headings above. I think what I meant by this questions is, if the margin above a heading is equal to the value for `--pf-c-content--body--MarginBottom` would I still provide a variable? I would think that we should.